### ISSUE TITLE
graalvm ffmpeg loader

### ISSUE THREAD

**Issue by jcfrk101** [bytedeco/javacpp-presets#933]
Hello I have been running the ffmpeg preset loader within micronauts just fine until I needed to compile the native binary for a google app engine deploy. Here is what I am doing 
```
String ffmpeg = Loader.load(org.bytedeco.ffmpeg.ffmpeg.class);
        ProcessBuilder pb = new ProcessBuilder(ffmpeg, "-i", downloadTemp.getAbsolutePath(), "-af", "loudnorm=I=-5:TP=-1.5:LRA=11:print_format=json", "-f", "null", "-");
        Process process = pb.redirectErrorStream(true).start();
```

As you can see I am using dynamic class loading, which is causing issues with graal, any ideas on how I can do the same thing without dynamic class loading?

**Comment by saudet** [comment]
It should work fine with GraalVM as well.

**Comment by jcfrk101** [comment]
Ok, well when running from a jar everything works fine.
When I compile a native binary I get a Class not found exception on this line.
```
String ffmpeg = Loader.load(org.bytedeco.ffmpeg.ffmpeg.class);
```

**Comment by saudet** [comment]
Please use 4.3.1-1.5.4-SNAPSHOT: http://bytedeco.org/builds/

**Comment by jcfrk101** [comment]
I switched to 4.3.1-1.5.4-SNAPSHOT and started getting build issues, specifically 
```
Caused by: java.lang.NoClassDefFoundError: kotlin/TypeCastException
```
after adding the kotlin standard library into my pom that error disappeared, but then I got a host of netty class not found errors followed by linking errors. It appears that the netty classes that where missing are JNI classes.

**Comment by saudet** [comment]
Right, that's unrelated to JavaCPP.

**Comment by saudet** [comment]
Support for GraalVM Native Image has been released with version 1.5.4.
